challenge_help_me_carry/src/challenge_help_me_carry.py

```diff
@@ -1,352 +1,150 @@
 #!/usr/bin/python
-import roslib;
+
 import rospy
 import smach
-import sys
-import time
-import datetime
-from hmi import TimeoutException
-
-from cb_planner_msgs_srvs.msg import PositionConstraint
-
-import robot_smach_states.util.designators as ds
-from robot_smach_states.util.designators import check_type
-from robot_skills.arms import Arm, GripperState
-
 import robot_smach_states as states
+import robot_smach_states.util.designators as ds
 
+from robot_skills.arms import Arm, GripperState
 from robocup_knowledge import load_knowledge
-from robot_skills.util import kdl_conversions
-from robot_skills.util.entity import Entity
+from challenge_hmc_functions import hmc_functions
 
 challenge_knowledge = load_knowledge('challenge_help_me_carry')
+common = load_knowledge('common')
```

Comment:
Add common knowledge to knowledge file by adding:
```
from robocup_knowledge import knowledge_loader
common = knowledge_loader.load_knowledge("common")
```
Then you can use challenge_knowledge.common

Reply:
Good idea 👍